NEWS

```diff
@@ -11,9 +11,156 @@
 ### Features:
 ### Bugfixes:
 
-## 1.18.0-rc2 (December 10, 2024)
-### Features: TBD
-### Bugfixes: TBD
+## 1.18.0-rc3 (December 18, 2024)
+### Feature
+#### UCP
+ * Enabled using CUDA staging buffers for pipeline protocols by default
+ * Added endpoint reconfiguration support for non-reused p2p scenarios
+ * Enabled non-cacheable memory domains, activated for GDR
+ * Added user_data parameter to ucp_ep_query
+ * Added support for host memory pipeline through CUDA buffers for rendezvous protocol
+ * Added global VA infrastructure and memory region in absence of error handling
+ * Made protocol performance node names more informative
+ * Enforced always running on the same thread in single thread mode
+ * Multiple improvements in protocols selection infrastructure
+ * Added UCP_MEM_MAP_BLOCK API flag to enforce blocking memory mapping
```

Comment:
Added UCP_MEM_MAP_LOCK API flag to enforce locked memory mapping

Reply:
fixed